### ISSUE TITLE
Create a single binary that installs daemons of every chain

### ISSUE THREAD

**Issue by sunnya97** [cosmos/chain-registry#106]

**Comment by marbar3778** [comment]
this is overkill and more so identifies a lack of tolling to use across chains. I would propose using lens from Strangelove instead. The tool provides a way to work with all chains without needing to download all binaries

**Comment by nooomski** [comment]
If anything, it would be nice to have something that generates a script to install a binary, and populate config files / address book based on the chain selected, so node operators can bootstrap more quickly.

**Comment by hxrts** [comment]
Feels like this goes without saying but any tooling like this should be in a separate repo.

**Comment by sunnya97** [comment]
I think this is effectively what lens is doing

https://github.com/strangelove-ventures/lens

@jackzampolin

**Comment by sunnya97** [comment]
Agree with @hxrts that this should be in a new repo, similar to lens and Akash's cosmos-omnibus https://github.com/ovrclk/cosmos-omnibus

**Comment by agouin** [comment]
I agree with the common sentiment. We are using [heighliner](https://github.com/strangelove-ventures/heighliner) for running containerized chain nodes, similar to [omnibus](https://github.com/ovrclk/cosmos-omnibus). For just about everything else, we use [lens](https://github.com/strangelove-ventures/lens).

**Comment by hxrts** [comment]
> how do you feel about docker-compose stuff living here?

My feeling is this whole repo should be parsable as linked json-ld. Don't have strong opinions about how deep that data structure should go, but it shouldn't really matter that much. We can always replace inline data with a link or visa-versa.
